lib/features/home/presentation/calentre_home.dart

```diff
@@ -38,24 +38,24 @@ class _CalentreHomeState extends State<CalentreHome> {
     );
   }
 
-  @override
-  void dispose() {
-    //close the CalentreEventBloc here
-    sl.get<CalentreEventBloc>().close();
-    super.dispose();
-  }
+  // @override
+  // void dispose() {
```

Comment:
if no needed would be good to remove comments

Reply:
Not needed for 👍